db/migrate/20181011094250_change_include_in_links_to_string.rb

```diff
@@ -0,0 +1,22 @@
+class ChangeIncludeInLinksToString < ActiveRecord::Migration[5.2]
+  def up
+    add_column :navigation_rules, :include_in_links_string, :string, default: 'always', null: false
+
+    NavigationRule.where(include_in_links: true).update_all(include_in_links_string: 'always')
```

Comment:
Given the column default, is this first `update_all` necessary?

Reply:
Nope, removed (also from the `down` method)